tfe/resource_tfe_variable.go

```diff
@@ -707,11 +722,46 @@ func (r *resourceTFEVariable) ImportState(ctx context.Context, req resource.Impo
 	resp.Diagnostics.Append(diags...)
 }
 
+type updateReadableValuePlanModifier struct{}
+
+func (u *updateReadableValuePlanModifier) Description(ctx2 context.Context) string {
```

Comment:
```suggestion
func (u *updateReadableValuePlanModifier) Description(ctx context.Context) string {
```

(And similar comment for the other uses of this below.)

I think it's probably better to use the conventional `ctx` name for these arguments... unless I'm missing something? 

We've still got a "default context" top-level `ctx` var hanging around in the package, and I notice that you're actually referencing _that_ object a few lines south of here when you reference `ctx` in the body of `PlanModifyString`. I'm guessing that's on purpose, but my understanding is: because these new framework functions can always rely on receiving an appropriate context from their caller, we don't need to mess around with having a top-scope "default context" at all — so it's actually good and proper for the function arguments to shadow the "default context" var that the sdkv2 provider relies on and just use whatever they got passed.

Reply:
You are totally right! This was an artifact of GoLand automatically completing the interface implementation for me, and I hadnt noticed. I will update it all to be `ctx`